data-prepper-plugins/anomaly-detector-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/anomalydetector/AnomalyDetectorProcessorConfig.java

```diff
@@ -34,14 +36,17 @@ public class AnomalyDetectorProcessorConfig {
 
     @JsonPropertyDescription("If provided, anomalies will be detected within each unique instance of these keys. For example, if you provide the ip field, anomalies will be detected separately for each unique IP address.")
     @JsonProperty("identification_keys")
+    @ExampleValues({
```

Comment:
I think we don't need an example here since this is just a key.

Reply:
Same here. I think for lists it doesn't hurt to have a placeholder value